app/views/proposal/new.html.haml

```diff
@@ -43,7 +43,7 @@
               :javascript
                 $("##{@program.event_types.first.id}-help").collapse('show');
 
-              = f.input :abstract, required: true, input_html: { rows: 5, data: { provide: 'markdown-editable' } },
+              = f.input :abstract, input_html: { rows: 5, data: { provide: 'markdown-editable' } },
```

Comment:
@ChrisBr do you think we should keep it with a condition? (If min_abstract_length == 0 not required, otherwise required)